apps/desktop-wallet/src/api/apiDataHooks/useAlphBalancesByAddress.ts

```diff
@@ -32,8 +32,8 @@ export interface AddressesAlphBalances {
   isLoading: boolean
 }
 
-const useAddressesAlphBalances = (addressHash?: AddressHash): AddressesAlphBalances => {
-  const { data: latestTxHashes, isLoading: isLoadingLatestTxHashes } = useAddressesLastTransactionHashes(addressHash)
+const useAlphBalancesByAddress = (): AddressesAlphBalances => {
```

Comment:
I find this name gives a better idea of what is returned (an object where keys are address hashes and values are balances)